pyproject.toml

```diff
@@ -132,6 +132,7 @@ ignore = [
   "D406", # Ignore new line after section name
   "D407", # Ignore dashed-underline-after-section
   "N803", # Argument casing
+  "N805", # Checks for instance methods that use a name other than self for their first argument.
```

Comment:
this is for cls method?

Reply:
In my local tests, it always show false alert on some instance method. Run linter runner init can fix.

Reply:
Just curious which method except classmethod.